src/poliastro/plotting.py

```diff
@@ -112,6 +112,14 @@ def _redraw(self):
         self.ax.relim()
         self.ax.autoscale()
 
+    def plot_trajectory(self, trajectory, color, label=None):
```

Comment:
This is inconsistent with `OrbitPlotter3D.plot_trajectory`:

https://github.com/poliastro/poliastro/pull/332/files#diff-be441b39adc37d0e0a9887217a253560R346

The signatures should be the same. That's the point of unifying the codebases, right? :wink:

Reply:
True!